README.md

```diff
@@ -100,6 +100,47 @@ var client = ads.connect(options, function() {
 })
 ```
 
+
+### MultiRead something
+
+```javascript
+        this.client.multiRead(
+            [{
+                symname: '.TESTBOOL',
+                indexGroup: 16448,
+                indexOffset: 10438,
+                bytelength: ads.BOOL,
+            }, {
+                symname: '.TESTINT',
+                indexGroup: 16448,
+                indexOffset: 26938,
+                bytelength: ads.UINT,
+            }],
+            function (err, result) {
+                if (err) {
+                    console.error(err)
+                }
+                console.log(result)
+            });
+```
+
+### Get handles
+
+```javascript
+        this.client.getHandles(
```

Comment:
I think the example should be like any other
```
var client = ads.connect(options, function() {
   client.getHandles(
....
}
```

Reply:
Yes i generally will update the readme today, it was very late yesterday.

Reply:
the "this.end ()" is also missing